dbms/tests/integration/test_mysql_protocol/clients/mysql/docker_compose.yml

```diff
@@ -1,6 +1,8 @@
 version: '2.2'
 services:
   mysql1:
-    image: mysql:5.7
-    # rewriting default command, because starting server is unnecessary
-    command: sleep infinity
+      image: mysql:5.7
+      restart: always
+      environment:
+          MYSQL_ALLOW_EMPTY_PASSWORD: 1
+      command: --federated --socket /var/run/mysqld/mysqld.sock
```

Comment:
Indent differs from an indent on line 3
Newline at the end of file is missing

Reply:
done